Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
+gem 'jazzy'
```

Comment:
I don't know enough about iOS development to know if this would be an issue cc @nakajima

Reply:
Seems fine to me.